swarm/storage/chunker_test.go

```diff
@@ -379,63 +275,54 @@ func benchReadAll(reader LazySectionReader) {
 func benchmarkJoin(n int, t *testing.B) {
 	t.ReportAllocs()
 	for i := 0; i < t.N; i++ {
-		chunker := NewTreeChunker(NewChunkerParams())
-		tester := &chunkerTester{t: t}
 		data := testDataReader(n)
 
-		chunkC := make(chan *Chunk, 1000)
-
-		key, wait, err := tester.Split(chunker, data, int64(n), chunkC, nil)
+		putGetter := newTestHasherStore(newTestChunkStore(), SHA3Hash)
+		key, wait, err := PyramidSplit(data, putGetter, putGetter)
 		if err != nil {
-			tester.t.Fatalf(err.Error())
+			t.Fatalf(err.Error())
 		}
 		wait()
-		chunkC = make(chan *Chunk, 1000)
-		quitC := make(chan bool)
-		reader := tester.Join(chunker, key, i, chunkC, quitC)
+		reader := TreeJoin(key, putGetter, 0)
```

Comment:
@nolash Your comment is valid, but since this is really not the scope of my PR to refactor the benchmark tests, I just renamed the function names, see c59a6a477bc8cd21fbbd79c96f60396cb30258dc
Is it better this way?